src/interface/meta.thrift

```diff
@@ -563,6 +567,9 @@ struct HBReq {
     5: binary     git_info_sha,
     // version of binary
     6: optional binary version,
+    7: optional map<common.GraphSpaceID, map<binary, PartitionList>
```

Comment:
why don't use list<common.PartitionID> directly ?

Reply:
PartitionList is the value in meta table. For simplicity, it's easy to serialize a thrift object. If use list<common.PartitionID> directly, I should to serialize the vector by my self.